nvtabular/inference/graph/ops/__init__.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# alias submodules here to avoid breaking everything with moving to submodules
+# flake8: noqa
```

Comment:
```suggestion
# alias submodules here to avoid breaking everything with moving to submodules
# flake8: noqa
```

Reply:
we need the flake8 noqa because of the method for imports... look at CI failures below I am going to revert this.